### PR TITLE
Adding Lambda Function URL configuration

### PR DESCRIPTION
**Pull request by astefanski-sumo**
As a follow-up to https://sumologic.zendesk.com/agent/tickets/81676 ZenDesk ticket I've updated the documentation with guidance to configure lambda webhook with Function URL.  On top of that I extracted IAM User creation and assigning role steps (to be done in AWS) from Create a Webhook connection section (to be done in Sumo)

## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions and updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/alerts/webhook-connections/aws-lambda.md [hunk 1]
```diff
@@ -13,11 +13,54 @@ You can trigger an AWS Lambda function directly from a Monitor or Scheduled Sear
...
-When you have created the Invoke URL, copy and paste it into a notepad. You will need it to configure the webhook connection in the next section.
+In more demanding use cases lambda can be accessed via **API Gateway**. To use this approach generate an Invoke URL, with a POST method for your Lambda function by creating an API in Amazon API Gateway. For information about exposing an HTTP endpoint in API Gateway, see Amazon's [API Gateway documentation](https://docs.aws.amazon.com/apigateway/latest/developerguide/getting-started.html).
+
+Secure your Lambda Function URL or API Gateway method by selecting **AWS_IAM** for the authorization type in configuration on AWS side.
+
+:::tip
+Having URL to the webhook, copy and paste it into a notepad. You will need it to configure the webhook connection in the next section.
+:::
+
+## Create an IAM user with required priviledges
```
Comment (kimsauce) [inline docs/alerts/webhook-connections/aws-lambda.md:28]:
```suggestion
## Create an IAM user with required privileges
```

File: docs/alerts/webhook-connections/aws-lambda.md [hunk 2]
```diff
@@ -13,11 +13,54 @@ You can trigger an AWS Lambda function directly from a Monitor or Scheduled Sear
...
+
+:::tip
+Having URL to the webhook, copy and paste it into a notepad. You will need it to configure the webhook connection in the next section.
+:::
+
+## Create an IAM user with required priviledges 
+
+### Lambda Function URL
+
+If you use Lambda Function URL create an IAM user with **lambda:InvokeFunctionUrl** action allowed.
```
Comment (kimsauce) [inline docs/alerts/webhook-connections/aws-lambda.md:32]:
```suggestion
If you use the Lambda Function URL, create an IAM user with **lambda:InvokeFunctionUrl** action allowed. 
```

File: docs/alerts/webhook-connections/aws-lambda.md [hunk 3]
```diff
@@ -34,11 +77,11 @@ Configure the webhook connection to trigger the AWS Lambda function.
...
     * **Description.** Optional: Enter a Description for the Connection.
     * **URL.** Enter the Invoke URL from the previous section.
-    * **Access Key and Secret Key.** Enter your AWS Access Key and Secret Key.
-    * Secure your API gateway method by selecting **AWS_IAM** for the authorization type.
-    * Create an IAM user who has basic API gateway invoke access. You can use the AWS managed policy **AmazonAPIGatewayInvokeFullAccess**.
+    * **Access Key ID** and **Secret Access Key.** Enter AWS Access key and Secret access key for the account with required IAM policy assigned created in previous section.
     * **Region.** Select your region.
-    * **Service Name.** Enter **execute-api** as the service name.
+    * **Service Name.** 
+      * For Lambda Function URL enter **lambda** as the service name
```
Comment (kimsauce) [inline docs/alerts/webhook-connections/aws-lambda.md:83]:
```suggestion
      * For Lambda Function URL enter **lambda** as the service name.
```

File: docs/alerts/webhook-connections/aws-lambda.md [hunk 4]
```diff
@@ -13,11 +13,54 @@ You can trigger an AWS Lambda function directly from a Monitor or Scheduled Sear
 
 For example, you can create a Monitor that triggers a Lambda function when too many requests are received from a suspicious IP address. The Lambda function can shut down additional requests from that IP address, while simultaneously sending a notification to the security team for review.
 
-## Build an API in the API Gateway to expose a Lambda function
+## Use Lambda Function URL or build an API in the API Gateway to expose a Lambda function
 
-First, generate an Invoke URL, with a POST method for your Lambda function by creating an API in Amazon API Gateway. For information about exposing an HTTP endpoint, see Amazon's [API Gateway documentation](https://docs.aws.amazon.com/apigateway/latest/developerguide/getting-started.html).
+Lambda can be called directly using **Function URL**. Check [Lambda Functions URLs documentation](https://docs.aws.amazon.com/lambda/latest/dg/urls-configuration.html) for details.
```
Comment (kimsauce) [inline docs/alerts/webhook-connections/aws-lambda.md:18]:
```suggestion
Lambda can be called directly using **Function URL**. See [Lambda Functions URLs documentation](https://docs.aws.amazon.com/lambda/latest/dg/urls-configuration.html) for details.
```

File: docs/alerts/webhook-connections/aws-lambda.md [hunk 5]
```diff
@@ -34,11 +77,11 @@ Configure the webhook connection to trigger the AWS Lambda function.
...
     * **URL.** Enter the Invoke URL from the previous section.
-    * **Access Key and Secret Key.** Enter your AWS Access Key and Secret Key.
-    * Secure your API gateway method by selecting **AWS_IAM** for the authorization type.
-    * Create an IAM user who has basic API gateway invoke access. You can use the AWS managed policy **AmazonAPIGatewayInvokeFullAccess**.
+    * **Access Key ID** and **Secret Access Key.** Enter AWS Access key and Secret access key for the account with required IAM policy assigned created in previous section.
     * **Region.** Select your region.
-    * **Service Name.** Enter **execute-api** as the service name.
+    * **Service Name.** 
+      * For Lambda Function URL, enter **lambda** as the service name.
+      * For API Gateway enter **execute-api** as the service name.
```
Comment (kimsauce) [inline docs/alerts/webhook-connections/aws-lambda.md:84]:
```suggestion
      * For API Gateway, enter **execute-api** as the service name.
```

File: docs/alerts/webhook-connections/aws-lambda.md [hunk 6]
```diff
@@ -13,11 +13,54 @@ You can trigger an AWS Lambda function directly from a Monitor or Scheduled Sear
...
+
+### Lambda Function URL
+
+If you use the Lambda Function URL, create an IAM user with **lambda:InvokeFunctionUrl** action allowed. 
+
+:::caution
+Required action is different than the *lambda:InvokeFunction* used in AWS managed IAM Policy *AWSLambdaRole* so one needs to create *Customer managed* role
+:::
+
+<details><summary>Template for the IAM Policy</summary>
```
Comment (kimsauce) [inline docs/alerts/webhook-connections/aws-lambda.md:38]:
```suggestion
<details><summary>IAM Policy template</summary>

```

File: docs/alerts/webhook-connections/aws-lambda.md [hunk 7]
```diff
@@ -13,11 +13,55 @@ You can trigger an AWS Lambda function directly from a Monitor or Scheduled Sear
...
+        }
+    ]
+}
+```
+
+</details>
+
+### API Gateway
+
+If you use API Gateway, create an IAM user who has basic API gateway invoke access. You can use the AWS managed policy **AmazonAPIGatewayInvokeFullAccess**.
```
Comment (kimsauce) [inline docs/alerts/webhook-connections/aws-lambda.md:60]:
```suggestion
If you use API Gateway, create an IAM user who has basic API gateway invoke access. You can use the AWS-managed policy **AmazonAPIGatewayInvokeFullAccess**.
```

File: docs/alerts/webhook-connections/aws-lambda.md [hunk 8]
```diff
@@ -13,11 +13,55 @@ You can trigger an AWS Lambda function directly from a Monitor or Scheduled Sear
...
+Lambda can be called directly using **Function URL**. See [Creating and managing Lambda function URLs](https://docs.aws.amazon.com/lambda/latest/dg/urls-configuration.html) for details.
 
-When you have created the Invoke URL, copy and paste it into a notepad. You will need it to configure the webhook connection in the next section.
+In more demanding use cases lambda can be accessed via **API Gateway**. To use this approach generate an Invoke URL, with a POST method for your Lambda function by creating an API in Amazon API Gateway. For information about exposing an HTTP endpoint in API Gateway, see Amazon's [API Gateway documentation](https://docs.aws.amazon.com/apigateway/latest/developerguide/getting-started.html).
+
+Secure your Lambda Function URL or API Gateway method by selecting **AWS_IAM** for the authorization type in configuration on AWS side.
+
+:::tip
+Having URL to the webhook, copy and paste it into a notepad. You will need it to configure the webhook connection in the next section.
```
Comment (kimsauce) [inline docs/alerts/webhook-connections/aws-lambda.md:25]:
```suggestion
Have your webhook URL handy by copying and pasting it to a notepad. You'll need it to configure your webhook connection in the next section.
```

File: docs/alerts/webhook-connections/aws-lambda.md [hunk 9]
```diff
@@ -13,11 +13,55 @@ You can trigger an AWS Lambda function directly from a Monitor or Scheduled Sear
...
+
+</details>
+
+### API Gateway
+
+If you use API Gateway, create an IAM user who has basic API gateway invoke access. You can use the AWS managed policy **AmazonAPIGatewayInvokeFullAccess**.
+
+:::tip
+Having user account with required IAM policy assigned create AWS Access Key for this account and note **Access key** and **Secret access key** to authenticate Webhook connection in next section.
+:::
```
[the remark this answers is not in the document]
Comment (astefanski-sumo) [reply docs/alerts/webhook-connections/aws-lambda.md:64]: Moving this from 'tip' to p2 for API Gateway section you'll need to copy the same for 'Function URL' section - in both cases one needs to have Access key and Secret access key handy.

File: docs/alerts/webhook-connections/aws-lambda.md [hunk 10]
```diff
@@ -13,54 +13,51 @@ You can trigger an AWS Lambda function directly from a Monitor or Scheduled Sear
 
 For example, you can create a Monitor that triggers a Lambda function when too many requests are received from a suspicious IP address. The Lambda function can shut down additional requests from that IP address, while simultaneously sending a notification to the security team for review.
 
-## Use Lambda Function URL or build an API in the API Gateway to expose a Lambda function
+## How to expose a Lambda function
+
+To expose a expose a Lambda function, you can either call a Lambda Function URL or build an API in the API Gateway.
```
Comment (astefanski-sumo) [inline docs/alerts/webhook-connections/aws-lambda.md:18]: Double "expose a" here. One to be removed?
"call a Lambda Function URL" needs to be replaced; I would rather say "enable Lambda Function URL"
image

File: docs/alerts/webhook-connections/aws-lambda.md [hunk 11]
```diff
@@ -13,54 +13,51 @@ You can trigger an AWS Lambda function directly from a Monitor or Scheduled Sear
...
+               "Effect": "Allow",
+               "Action": [
+                   "lambda:InvokeFunctionUrl"
+               ],
+               "Resource": "arn:aws:lambda:*:*:function:*"
+           }
+       ]
+   }
+   ```
+3. Create an AWS Access Key for this account. Take note of your **Access key** and **Secret access key**, which you'll need in the next section to authenticate your Webhook connection.<br/><img src={useBaseUrl('img/connection-and-integration/customer-managed-policy.png')} alt="Customer managed policy" />
```
Comment (astefanski-sumo) [inline docs/alerts/webhook-connections/aws-lambda.md:53]: I've shared picture to give you some context. Not sure if we should include this in our doc.

File: docs/alerts/webhook-connections/aws-lambda.md [hunk 12]
```diff
@@ -13,11 +13,51 @@ You can trigger an AWS Lambda function directly from a Monitor or Scheduled Sear
 
 For example, you can create a Monitor that triggers a Lambda function when too many requests are received from a suspicious IP address. The Lambda function can shut down additional requests from that IP address, while simultaneously sending a notification to the security team for review.
 
-## Build an API in the API Gateway to expose a Lambda function
+## How to expose a Lambda function
 
-First, generate an Invoke URL, with a POST method for your Lambda function by creating an API in Amazon API Gateway. For information about exposing an HTTP endpoint, see Amazon's [API Gateway documentation](https://docs.aws.amazon.com/apigateway/latest/developerguide/getting-started.html).
+To expose a expose a Lambda function, you can either call a Lambda Function URL or build an API in the API Gateway.
```
Comment (kimsauce) [inline docs/alerts/webhook-connections/aws-lambda.md:18]:
```suggestion
To expose a Lambda function, you can use the **Enable function URL** option or build an API in the API Gateway.
```

**Review by kimsauce — APPROVED** [review]

**Pull request merged**